smartsim/_core/generation/generator.py

```diff
@@ -136,10 +145,17 @@ def generate_job(self, job: Job, job_index: int) -> pathlib.Path:
             dt_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
             log_file.write(f"Generation start date and time: {dt_string}\n")
 
+        # Create output files
+        out_file, err_file = self._output_files(log_path, job.entity.name)
+        # Open and write to .out file
+        open(out_file, mode="w", encoding="utf-8")
+        # Open and write to .err file
+        open(err_file, mode="w", encoding="utf-8")
```

Comment:
Is there a reason we open these files but then never close (or hold a reference to) the descriptor? Can these be removed?

Reply:
Is this just here to so that the file exists? If so, consider using `pathlib.Path.touch`

Reply:
I believe I was just trying to get the file in the dir

Reply:
but its opened later, I removed